### ISSUE TITLE
Bug: hud_quickinfo_swap 1 crosshair swaps the monoportal crosshair color

### ISSUE THREAD

**Issue by lenship2** [StrataSource/Portal-2-Community-Edition#1780]
### Describe the bug

I personally use hud_quickinfo_swap 1 in my daily Portal 2 routines, and one thing that has always bothered me is the fact that the command will switch the crosshair when there's only one portal, meaning that a blue only portal gun will have an orange crosshair, and vice versa. While it's only a visual bug, it can be annoying to look at and leads to a lot of people asking why I'm playing an 'orange monoportal map'.

![Portal 2_ Community Edition - DX11 12_21_2024 2_24_29 PM](https://github.com/user-attachments/assets/d2ce7707-20a3-4193-8b14-d537f70f2c39)
> Smooth Jazz (sp_a1_into4), with the command enabled.

### Issue Map

sp_a1_intro3 to sp_a1_wakeup

### To Reproduce

1. Go into any sp_a1 map past sp_a1_intro3
2. Open the console and enter the command hud_quickinfo_swap 1
3. shoot blue portal with orange crosshair (pls fix)

**Comment by lenship2** [comment]
I think with the implementation of customizable portal colors, this issue can be closed. Anyone who wants to swap the portal colors can do so indirectly through portal color customization

**Comment by TeamSpen210** [comment]
Not really, the portals would be swapped then, so you're effectively playing an orange-portal map.
